sdk/tests/AccountBalanceQueryTest.cc

```diff
@@ -0,0 +1,72 @@
+/*-
+ *
+ * Hedera C++ SDK
+ *
+ * Copyright (C) 2020 - 2022 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
```

Comment:
It might make sense at some point to make stubby test Clients / Nodes, to be able to test functions like `getGrpcMethod`, `makeRequest`, and `mapResponse`. Thoughts?

Reply:
Yep absolutely. We can probably use `gmock` for that, I think it lets you stub classes out like that. I'd need to look into how exactly to set it up, but I think that'd be the path forward with that.